app/server/api/user_api.py

```diff
@@ -210,10 +210,13 @@ def post(self, user_id):
 
         if reset_user_id is not None:
```

Comment:
this endpoint is about to become a lot more critical - would be goood to have a test for it

Reply:
Beefed up the test for this a bit more!